packages/forms/src/UIForm/UIForm.component.js

```diff
@@ -90,25 +91,51 @@ export default class UIForm extends React.Component {
 	 * @param event the submit event
 	 */
 	submit(event) {
-		event.preventDefault();
+		if (this.props.onSubmit) {
+			event.preventDefault();
+		}
+
 		const { mergedSchema } = this.state;
 		const { formName, properties, customValidation } = this.props;
 		const errors = validateAll(mergedSchema, properties, customValidation);
 		this.props.setErrors(formName, errors);
 
 		const isValid = !Object.keys(errors).length;
-		if (isValid) {
+		if (this.props.onSubmit && isValid) {
 			this.props.onSubmit(event, properties);
 		}
+
+		return isValid;
 	}
 
 	render() {
-		const { autoComplete, errors, formName, id, properties, widgets } = this.props;
+		const {
+			acceptCharset,
+			action,
+			autoComplete,
+			encType,
+			formName,
+			id,
+			method,
+			noHtml5Validate,
+			target,
+
+			errors,
+			properties,
+			widgets,
+		} = this.props;
 		return (
 			<form
-				onSubmit={this.submit}
+				acceptCharset={acceptCharset}
+				action={action}
 				autoComplete={autoComplete}
-				noValidate
+				encType={encType}
+				id={id}
+				method={method}
+				name={formName}
+				noValidate={noHtml5Validate}
+				onSubmit={this.submit}
+				target={target}
```

Comment:
I don't see why you would need to spread so many values (creating a ton of allocations) simply to avoid using the `this.props.` notation.